install/kubernetes/helm/istio-remote/values.yaml

```diff
@@ -34,6 +34,9 @@ global:
   # Default tag for Istio images.
   tag: master-latest-daily
 
+  # monitoring port used by mixer, pilot, galley
```

Comment:
we should hardcode the port.  We do that for the rest of Istio.

Reply:
(there was great debate about this in the past as well, and we decided on the hardcodes).

Reply:
I removed using a value for the monitoring port in istio-remote that I had added.
I left the value in the istio branch (just changed the value).